README.md

```diff
@@ -267,7 +269,8 @@ or previous function.
 lua <<EOF
 require'nvim-treesitter.configs'.setup {
   textobjects = {
-    move {
+    enable = true,
+    move = {
```

Comment:
```suggestion
    move = {
```

Reply:
This also needs to be fixed in https://github.com/nvim-treesitter/nvim-treesitter/blob/7f81df44098a7fef93d6901a06b8312c5b8619d7/doc/nvim-treesitter.txt#L365-L365